src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HPackEncoder.cs

```diff
@@ -20,6 +23,33 @@ internal static class HPackEncoder
         //          So for now, don't do dynamic encoding.
 
         public static bool EncodeHeader(string name, string value, Span<byte> buffer, out int length)
+        {
+            Debug.Assert(value != null);
+            return EncodeHeader(name, value, null, null, buffer, out length);
+        }
+
+        public static bool EncodeHeader(Headers.HeaderDescriptor header, string value, Span<byte> buffer, out int length)
+        {
+            return EncodeHeader(header.Name, value, null, null, buffer, out length);
+        }
+
+        public static bool EncodeHeader(Headers.HeaderDescriptor header, string[] values, string separator, Span<byte> buffer, out int length)
+        {
+            return EncodeHeader(header.Name, values, separator, buffer, out length);
+        }
+
+        public static bool EncodeHeader(string name, string[] values, string separator, Span<byte> buffer, out int length)
+        {
```

Comment:
Debug.Assert(values != null), to match the similar check for value in the other overload?